x-pack/plugin/esql/qa/testFixtures/src/main/resources/data/dense_vector.csv

```diff
@@ -1,5 +1,5 @@
-id:l, vector:dense_vector
-0,    [1.0, 2.0, 3.0]
-1,    [4.0, 5.0, 6.0]
-2,    [9.0, 8.0, 7.0]
-3,    [0.054, 0.032, 0.012]
+id:l, float_vector:dense_vector, byte_vector:dense_vector
```

Comment:
what is `id:l` ? is this right? or a typo?

Reply:
Looks like a typo, thanks!